### PR TITLE
Avatar and badge components

### PR DESCRIPTION
**Pull request by kart-c**

### CODE REVIEW

File: Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="avatar.css" />
+		<title>Avatar</title>
+	</head>
+	<body>
+		<h1>Avatar Component</h1>
+		<img src="../Assets/avatar-1.jpg" alt="" class="avatar avatar-lg" />
```
Comment (VishalPatil18) [inline Avatar/avatar.html]: **FEAT:** Give a value to "alt" attribute as it helps for accessibility.

**Review by VishalPatil18 — APPROVED** [review]
Everything looks good. Just one suggestion to give values to the "alt" attribute as it can help for accessibility. Rest well done 👍

File: Avatar/avatar.html (hunk 1 above)
Comment (kart-c) [reply Avatar/avatar.html]: thank you Vishal. I will change it right away

File: Badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script src="https://kit.fontawesome.com/bbbaa28fd5.js" crossorigin="anonymous"></script>
+		<link rel="stylesheet" href="../Avatar/avatar.css" />
+		<link rel="stylesheet" href="badge.css" />
+		<title>Badge</title>
+	</head>
+	<body>
+		<h1>Badge on Avatar</h1>
+		<div class="badge-container">
+			<img src="../Assets/avatar-2.jpg" alt="user avatar" class="avatar avatar-lg" />
```
Comment (RakshaPawar108) [inline Badge/badge.html:15]: I am not able to view Badge on Avatar images in CSB. Please check. Maybe you have added your assets folder in gitignore so it is giving an error in console probably.

Comment (RakshaPawar108) [reply Badge/badge.html:15]: Same with Avatar component.

**Comment by AnkurChunekar** [comment]
Hey, your components look great, but the images do not load and due to that the badges are distorted can you look into this.
kartik review

File: Badge/badge.html (hunk 2 above)
Comment (kart-c) [reply Badge/badge.html:15]: Thank you for the review Raksha, I removed the file from `.gitignore`, can you check it now?

**Comment by kart-c** [comment]
> Hey, your components look great, but the images do not load and due to that the badges are distorted can you look into this. kartik review


Thanks Ankur I will fix it right away

**Comment by RakshaPawar108** [comment]
Looks good now Kartikeya! Great work on the components. Very clean code too

**Review by RakshaPawar108 — APPROVED** [review]

**Pull request merged**